### PR TITLE
Reorganise content around new local analysis features

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
### Proposed changes

Add and reorganise content based on new local features for scanning artifactory images locally.

### CODE REVIEW

File: scout/artifactory.md [hunk 1]
```diff
@@ -20,13 +20,25 @@ title: Artifactory integration
 Integrating Docker Scout with JFrog Artifactory lets you run image analysis
 automatically on images in your Artifactory registries.
 
-This integration is made possible by a monitoring agent. The agent is a
+## Local image analysis
+
+You can analyze Artifactory images for vulnerabilities locally using Docker Desktop or the Docker CLI. You first need to authenticate with JFrog Artifactory using the `[Docker login](/engine/reference/commandline/login/)` command. For example:
+
+```bash
+docker login {REPOSITORY_URL}.jfrog.io
```
Comment (chrispatrick) [inline scout/artifactory.md:28]: It won't necessarily end in `jfrog.io` for instance if the user has setup their own self-hosted instance.

File: scout/artifactory.md [hunk 2]
```diff
@@ -20,13 +20,25 @@ title: Artifactory integration
...
+```
+
+You can find the credentials for your Artifactory repository by selecting it in the Artifactory UI and then the **Set Me Up** button.
+
+## Remote image analysis
+
+To automatically analyze images and containers running in remote environments you need to deploy the Docker Scout Artifactory agent. The agent is a
```
Comment (chrispatrick) [inline scout/artifactory.md:35]: Technically we aren't analyzing containers, just images.

File: scout/artifactory.md (hunk 1 above)
Comment (ChrisChinchilla) [reply scout/artifactory.md:28]:
```suggestion
docker login {REPOSITORY_URL}
```

File: scout/artifactory.md (hunk 2 above)
Comment (ChrisChinchilla) [reply scout/artifactory.md:35]:
```suggestion
To automatically analyze images running in remote environments you need to deploy the Docker Scout Artifactory agent. The agent is a
```

File: scout/index.md [hunk 3]
```diff
@@ -66,7 +66,7 @@ For more information, see
 ### Artifactory integration
 
 Users of JFrog Artifactory, or JFrog Container Registry, can integrate Docker
-Scout to enable automatic analysis of images. For more information, see
+Scout to enable automatic analysis of images locally and in production. For more information, see
```
Comment (chrispatrick) [inline scout/index.md:69]: Not sure if `in production` is accurate enough. We can analyze in any Artifactory instance which might be production but might equally be staging or test or whatever.

File: scout/artifactory.md [hunk 4]
```diff
@@ -20,13 +20,25 @@ title: Artifactory integration
...
-This integration is made possible by a monitoring agent. The agent is a
+## Local image analysis
+
+You can analyze Artifactory images for vulnerabilities locally using Docker Desktop or the Docker CLI. You first need to authenticate with JFrog Artifactory using the `[Docker login](/engine/reference/commandline/login/)` command. For example:
+
+```bash
+docker login {REPOSITORY_URL}.jfrog.io
+```
+
+You can find the credentials for your Artifactory repository by selecting it in the Artifactory UI and then the **Set Me Up** button.
```
[the remark this answers is not in the document]
Comment (ChrisChinchilla) [reply scout/artifactory.md:31]: Hmm, hard for me to test the other one, I'll see if there's anything in their docs

File: scout/artifactory.md (hunk 1 above)
Comment (chrispatrick) [reply scout/artifactory.md:28]: I think `REPOSITORY_URL` might not be ideal. Maybe `ARTIFACTORY_URL` or `INSTANCE_URL`?

File: scout/index.md (hunk 3 above)
Comment (ChrisChinchilla) [reply scout/index.md:69]:
```suggestion
Scout to enable automatic analysis of images locally and remotely. For more information, see
```

Sorry that should have been changed already.

File: scout/artifactory.md (hunk 1 above)
Comment (chrispatrick) [reply scout/artifactory.md:28]: OK sure. I just wasn't sure because Artifactory repository != Docker repository... 😬

Comment (ChrisChinchilla) [reply scout/artifactory.md:28]: hehe, yeah

**Review by chrispatrick — APPROVED** [review]
LGTM :shipit:

**Review by dvdksn — APPROVED** [review]

**Pull request merged**